src/planner/macos/mod.rs

```diff
@@ -364,6 +368,40 @@ fn check_not_running_in_rosetta() -> Result<(), PlannerError> {
     Ok(())
 }
 
+async fn check_suis() -> Result<(), PlannerError> {
+    let policies: profiles::Policies = match profiles::load().await {
+        Ok(pol) => pol,
+        Err(e) => {
+            tracing::info!(
+                "Error loading profile data: {:?}, skipping SystemUIServer checks.",
```

Comment:
IMHO:

1. This should be a `warn` line to distinguish from the standard `info` logging we do
2. My suggested wording makes it possible to put the error at the end (so it's not sandwiched between other info that's likely not as useful as seeing what actually went wrong), but I'm not attached as long as the actual error goes at the end

```suggestion
            tracing::warn!(
                "Skipping SystemUIServer checks: failed to load profile data: {:?}",
```